dev-docs/design/ssr-react.md

```diff
@@ -0,0 +1,96 @@
+# Server Rendering Lit Components in React
+
```

Comment:
Is it worth dating this document?

Reply:
I had considered that but figured the git history would serve that purpose, especially as this doc gets updated over time.